main/src/main/scala/sbt/internal/AnalysisUtil.scala

```diff
@@ -34,7 +33,9 @@ private[sbt] object AnalysisUtil {
       useConsistent = useConsistent,
       mappers = ReadWriteMappers.getEmptyMappers(),
       sort = true,
-      ec = ExecutionContext.global,
+      ec = scala.concurrent.ExecutionContext.global,
+      // reproducisble = true will wipe out the timestamp, which we need for sbt 1.x
```

Comment:
```suggestion
      // reproducible = true will wipe out the timestamp, which we need for sbt 1.x
```

Reply:
Addressed in 12db63951b9a34c196622dbf7b8a7fdfe679435a